ops/testing.py

```diff
@@ -3204,6 +3204,44 @@ def add_layer(
                         s._merge(service)
                     else:
                         layer.services[name] = service
+            for name, check in layer_obj.checks.items():
+                if not check.override:
+                    raise RuntimeError(
+                        f'500 Internal Server Error: layer "{label}" must define'
+                        f'"override" for check "{name}"'
+                    )
+                if check.override not in ('merge', 'replace'):
+                    raise RuntimeError(
+                        f'500 Internal Server Error: layer "{label}" has invalid '
+                        f'"override" value on check "{name}"'
```

Comment:
The Pebble error message is actually "value for check" rather than "value on check". Could you please update these and the existing service one too?